data/mods/Aftershock/maps/overmap_terrain.json

```
@@ -234,5 +234,69 @@
     "sym": "<",
     "color": "green",
     "see_cost": 3
+  },
+  {
+    "type": "overmap_terrain",
+    "id": [
+      "arcology_interior_center_robots",
+      "arcology_interior_w_side_robots",
+      "arcology_interior_e_side_robots",
+      "arcology_interior_n_side_robots",
+      "arcology_interior_s_side_robots",
+      "arcology_interior_ne_corner_robots",
+      "arcology_interior_nw_corner_robots",
+      "arcology_interior_se_corner_robots",
+      "arcology_interior_sw_corner_robots",
+      "arcology_exterior_e_side_robots",
+      "arcology_exterior_n_side_robots",
+      "arcology_exterior_s_side_robots",
+      "arcology_exterior_w_side_robots",
+      "arcology_exterior_ne_corner_robots",
+      "arcology_exterior_nw_corner_robots",
+      "arcology_exterior_se_corner_robots",
+      "arcology_exterior_sw_corner_robots",
+      "arcology_exterior_n_entrance_robots",
+      "arcology_exterior_s_entrance_robots",
+      "arcology_exterior_e_entrance_robots",
+      "arcology_exterior_w_entrance_robots"
+    ],
+    "name": "Arcology Ground Level (no life signs)",
```

Sort of feel these break the 4th wall a bit. Maybe just name it
```suggestion
    "name": "Desolate Arcology - Ground Level",
```